utils/include/comm/Connection.h

```diff
@@ -54,15 +54,19 @@ class Connection
 
     void shutdown();
 
+    void set_buffer_size_limit(uint32_t buffer_size_limit);
+
 protected:
 
     const unsigned MAX_PORT_NUMBER  = 65535;
     const unsigned MAX_RETRIES      = 100;
-    const unsigned MAX_BUFFER_SIZE  = (32*1024*1024);
+    const unsigned DEFAULT_BUFFER_SIZE  = (32*1024*1024);  // VDMS ORIGINAL DEFAULT SIZE
+    const unsigned MAX_BUFFER_SIZE  = (1024*1024*1024);  // 256MB for ATTD & 1GB Video Curation
```

Comment:
Could you remove these comment?
```suggestion
    const unsigned MAX_BUFFER_SIZE  = (1024*1024*1024);
```